### ISSUE TITLE
Full Path

### ISSUE THREAD

**Issue by SinewaveLaboratorium** [brianpetro/obsidian-smart-connections#448]
Hey there! Love the plugin.

With the latest updates, the feature "Show Full Path" has stopped working like it used to. More specifically, turning it off keeps pointing to subsections of my notes, instead of simply the title. Could you look into it?

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/125192779/5eec2180-2a57-4313-87a6-adfd72ecd780)

**Comment by brianpetro** [comment]
Hey @SinewaveLaboratorium  😊

Are trying to only see the name of the note? If so, you could set the block embedding model to "None".

The show full path should only toggle the folder path, so what you're seeing might be the correct functionality.

Let me know if that makes sense.

🌴

**Comment by SinewaveLaboratorium** [comment]
Yeah, I wanted to see only the name of the note, which would also avoid showing the same note multiple times (e.g. in the image above, Echoic Memory appears mentioned twice). In version 1 there was an option that prevented this. I thought it was _Show Full Path_ but I might be confusing it with other option. Regardless, seems like I can't get it to show it like that anymore 🤔

**Comment by brianpetro** [comment]
There was a feature that grouped blocks with the same note.

Do you think it would make sense to turn off the blocks in your settings? Then it would just show the notes, no heading-specific results.

🌴

**Comment by SinewaveLaboratorium** [comment]
Okay, I see why it didn't work the first time — I was turning off the notes embedding, not the blocks. Now it's working, thanks man! 🙂

By the way, am I missing any other functionality by leaving the blocks embedding off?

**Comment by brianpetro** [comment]
> By the way, am I missing any other functionality by leaving the blocks embedding off?

Blocks are kind of like a more granular note. They exist to improve performance in vaults with long notes and lots of headings.

So vaults with short notes and few headings will be fine without blocks.

🌴
